sdk/computefleet/Azure.ResourceManager.ComputeFleet/samples/Generated/Samples/Sample_ComputeFleetCollection.cs

```diff
@@ -24,7 +24,7 @@ public partial class Sample_ComputeFleetCollection
         public async Task CreateOrUpdate_FleetsCreateOrUpdate()
         {
             // Generated from example definition: 2024-11-01/Fleets_CreateOrUpdate.json
-            // this example is just showing the usage of "Fleet_CreateOrUpdate" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "Fleets_createOrUpdate" operation, for the dependent resources, they will have to be created separately.
```

Comment:
This is generated from pure tsp. Operation name in tsp is `createOrUpdate`: https://github.com/Azure/azure-rest-api-specs/blob/95aa73b8831deb2940ac7067b8f296cd189a5fe1/specification/azurefleet/AzureFleet.Management/fleet.tsp#L533

Reply:
same as other operations